lnd_test.go

```diff
@@ -1792,34 +1820,48 @@ func testRevokedCloseRetribution(net *networkHarness, t *harnessTest) {
 	// broadcasting his current channel state. This is actually the
 	// commitment transaction of a prior *revoked* state, so he'll soon
 	// feel the wrath of Alice's retribution.
-	breachTXID := closeChannelAndAssert(ctxb, t, net, net.Bob, chanPoint,
-		true)
+	force := true
+	closeUpdates, _, err := net.CloseChannel(ctxb, net.Bob, chanPoint, force)
+	if err != nil {
+		t.Fatalf("unable to close channel: %v", err)
+	}
 
-	// Query the mempool for Alice's justice transaction, this should be
-	// broadcast as Bob's contract breaching transaction gets confirmed
-	// above.
-	var justiceTXID *chainhash.Hash
-	breakTimeout := time.After(time.Second * 5)
-poll:
-	for {
-		select {
-		case <-breakTimeout:
-			t.Fatalf("justice tx not found in mempool")
-		default:
-		}
+	// Wait for Bob's breach transaction to show up in the mempool to ensure
+	// that Alice's node has started waiting for confirmations.
+	_, err = waitForTxInMempool(net.Miner.Node, 5*time.Second)
+	if err != nil {
+		t.Fatalf("unable to find Bob's breach tx in mempool: %v", err)
+	}
+	time.Sleep(100 * time.Millisecond)
```

Comment:
Why's this sleep needed? I suspect it's to give Alice enough time to write the breach state to disk, if so a comment should be added. 

Actually, I think if my suggestion above to also examine the partially sent breach messages on start up, then we won't need this.